### PR TITLE
Added fish git repository greeter script to wiki

### PR DESCRIPTION
**Pull request by TheSast**
Preview [here](https://github.com/TheSast/onefetch/wiki/getting-started)

### CODE REVIEW

File: .github/wiki/Getting-Started.md [hunk 1]
```diff
@@ -44,6 +44,25 @@ cd() {
 check_directory_for_new_repository
 ```
 
+By @TheSast
+
+A fish adaptation of the previous script to put in your `$XDG_CONFIG_HOME/fish/functions/cd.fish`:
```
Comment (spenserblack) [inline .github/wiki/Getting-Started.md:49]: Couldn't you use fish's built-in `funcsave` to save the functions to the correct path for each OS?

File: .github/wiki/Getting-Started.md [hunk 2]
```diff
@@ -44,6 +44,25 @@ cd() {
...
+```fish
+function cd -w='cd'
+  builtin cd $argv || return
+  check_directory_for_new_repository
+end
+
+function check_directory_for_new_repository
+  set current_repository (git rev-parse --show-toplevel 2> /dev/null)
```
Comment (spenserblack) [inline .github/wiki/Getting-Started.md:57]: Should these be `set --local`?

**Review by spenserblack — COMMENTED** [review]
As a fish user, this is much appreciated!

File: .github/wiki/Getting-Started.md [hunk 3]
```diff
@@ -44,6 +44,25 @@ cd() {
...
+  if [ "$current_repository" ] && \
+    [ "$current_repository" != "$last_repository" ]
+    onefetch
+  end
+  set -x last_repository $current_repository
```
Comment (o2sh) [inline .github/wiki/Getting-Started.md:62]:
```suggestion
  set -gx last_repository $current_repository
```

When I tested locally, it seems to always run `onefetch` when navigating inside a repository.

To fix this issue, I added the -g flag to the set command to mark last_repository as a global variable, so its value is remembered between function calls

File: .github/wiki/Getting-Started.md (hunk 2 above)
Comment (TheSast) [reply .github/wiki/Getting-Started.md:57]: The variable is local to the currently executing function, I think `--local ` is used to restrict scope to the innermost block.
 AFAIK this should only be an issue if this was not inside a function, which it is,

File: .github/wiki/Getting-Started.md (hunk 3 above)
Comment (TheSast) [reply .github/wiki/Getting-Started.md]: Thanks for catching the typo!

File: .github/wiki/Getting-Started.md (hunk 1 above)
Comment (TheSast) [reply .github/wiki/Getting-Started.md]: I didn't think of that, I could and I should.
[ I did as you suggested](https://github.com/TheSast/onefetch/commit/b27e54080faa3a5e60f7ef7621d737000c77750a)

**Review by spenserblack — APPROVED** [review]

**Review by o2sh — APPROVED** [review]
Thanks a lot for your first contribution @TheSast 🎉 and to @spenserblack for taking the time to review it 😊

**Pull request merged**